### ISSUE TITLE
Ember 1.13.10 get('content') vs toArrray()

### ISSUE THREAD

**Issue by kielni** [hhff/ember-infinity#111]
https://github.com/hhff/ember-infinity/blob/master/addon/mixins/route.js#L315 calls `newObjects.get('content')` to add new objects.  But in Ember 1.13.10, this returns an array of `InternalModel` objects, not model classes.  

From https://github.com/emberjs/data/issues/3604, it looks like this should be `newObjects.toArray()` instead.  I don't know the history, so it's hard to tell what that might break.  Suggestions?

**Comment by kellyselden** [comment]
Good find. It could be updated to do a ember data version check, then do the appropriate action.

**Comment by kielni** [comment]
I tried returning `newObjects.toArray()`  and it broke the tests all over:

    not ok 3 PhantomJS 1.9 - Acceptance: Infinity Route: it works when embedded route is refreshed
    TypeError: 'undefined' is not a function (evaluating 'internalModel.getRecord()')

I saw https://github.com/emberjs/data/issues/3530, and tried `return infinityModel.get('content').pushObjects(content);` but that broke tests too:

    not ok 48 PhantomJS 1.9 - RouteMixin - reaching the end: it uses extra params when loading more data
    'undefined' is not an object (evaluating 'infinityModel.get('content').pushObjects')

I don't understand Ember Data internals well enough to tell what should be happening. http://emberjs.com/api/data/classes/DS.RecordArray.html says "The record array materializes records as needed when they are retrieved for the first time.", but that doesn't seem to be happening.  Am I missing something obvious for getting this working with 1.13.10?
